package.json

```diff
@@ -4,9 +4,17 @@
   "main": "index.js",
   "repository": "git@github.com:ibexa/documentation-developer.git",
   "scripts": {
-     "scss": "node-sass --watch scss -o docs/css"
+    "scss": "sass --watch scss:docs/css",
+    "test": "yarn prettier-test && yarn eslint-test",
+    "fix": "yarn prettier-test --write && yarn eslint-test --fix",
+    "eslint-test": "eslint \"./code_samples/**/*.js\"",
+    "prettier-test": "yarn prettier \"./code_samples/**/*.{js,scss}\" --check"
+  },
+  "prettier": "eslint-config-ibexa/prettier",
+  "devDependencies": {
+    "eslint-config-ibexa": "https://github.com/ibexa/eslint-config-ibexa.git#v1.0.0"
```

Comment:
In 4.6 we use ~v1.1.1 and in 5.0 we have ~v1.2.0. I'm not sure if you want to have v1.0.0 here.

Reply:
I will update, thanks!

Reply:
Bumped in https://github.com/ibexa/documentation-developer/pull/2573/commits/faf80a2fce0339770b8b51498430e2473c2137f7